Jellyfin.Api/Controllers/ChannelsController.cs

```diff
@@ -0,0 +1,251 @@
+        /// </summary>
+        /// <response code="200">All channel features returned.</response>
+        /// <returns>An <see cref="OkResult"/> containing the channel features.</returns>
+        [HttpGet("Features")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public ActionResult<IEnumerable<ChannelFeatures>> GetAllChannelFeatures()
+        {
+            return _channelManager.GetAllChannelFeatures();
+        }
+
+        /// <summary>
+        /// Get channel features.
+        /// </summary>
+        /// <param name="id">Channel id.</param>
+        /// <response code="200">Channel features returned.</response>
+        /// <returns>An <see cref="OkResult"/> containing the channel features.</returns>
+        [HttpGet("{Id}/Features")]
+        public ActionResult<ChannelFeatures> GetChannelFeatures([FromRoute] string id)
+        {
+            return _channelManager.GetChannelFeatures(id);
+        }
+
+        /// <summary>
+        /// Get channel items.
+        /// </summary>
+        /// <param name="id">Channel Id.</param>
+        /// <param name="folderId">Optional. Folder Id.</param>
+        /// <param name="userId">Optional. User Id.</param>
+        /// <param name="startIndex">Optional. The record index to start at. All items with a lower index will be dropped from the results.</param>
+        /// <param name="limit">Optional. The maximum number of records to return.</param>
+        /// <param name="sortOrder">Optional. Sort Order - Ascending,Descending.</param>
+        /// <param name="filters">Optional. Specify additional filters to apply. This allows multiple, comma delimited. Options: IsFolder, IsNotFolder, IsUnplayed, IsPlayed, IsFavorite, IsResumable, Likes, Dislikes.</param>
+        /// <param name="sortBy">Optional. Specify one or more sort orders, comma delimited. Options: Album, AlbumArtist, Artist, Budget, CommunityRating, CriticRating, DateCreated, DatePlayed, PlayCount, PremiereDate, ProductionYear, SortName, Random, Revenue, Runtime.</param>
+        /// <param name="fields">Optional. Specify additional fields of information to return in the output. This allows multiple, comma delimited. Options: Budget, Chapters, DateCreated, Genres, HomePageUrl, IndexOptions, MediaStreams, Overview, ParentId, Path, People, ProviderIds, PrimaryImageAspectRatio, Revenue, SortName, Studios, Taglines.</param>
+        /// <response code="200">Channel items returned.</response>
+        /// <returns>Channel items.</returns>
+        [HttpGet("{Id}/Items")]
+        public async Task<ActionResult<QueryResult<BaseItemDto>>> GetChannelItems(
+            [FromRoute] Guid id,
+            [FromQuery] Guid? folderId,
+            [FromQuery] Guid? userId,
+            [FromQuery] int? startIndex,
+            [FromQuery] int? limit,
+            [FromQuery] string sortOrder,
+            [FromQuery] string filters,
+            [FromQuery] string sortBy,
+            [FromQuery] string fields)
```

Comment:
Calling this method without any parameters specified returns a 400 with this response:

```json
{"type":"https://tools.ietf.org/html/rfc7231#section-6.5.1","title":"One or more validation errors occurred.","status":400,"traceId":"|d8d39afd-4b07ea9dd09cb2da.","errors":{"fields":["The fields field is required."],"filters":["The filters field is required."],"channelIds":["The channelIds field is required."]}}
```

Reply:
Note: if sending an invalid ChannelId you'll still get a 400 response "Value cannot be null. (Parameter 'channel')"

Emby's way of returning a 404